### ISSUE TITLE
Failed to activate the atom-speech-to-code package

### ISSUE THREAD

**Issue by imsaar** [jedahan/atom-speech-to-code#2]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.210.0
**System**: Mac OS X 10.10.3
**Thrown From**: [atom-speech-to-code](https://github.com/jedahan/atom-speech-to-code) package, v0.3.0
### Stack Trace

Failed to activate the atom-speech-to-code package

```
At Cannot read property 'command' of undefined

TypeError: Cannot read property 'command' of undefined
    at Object.module.exports.activate (/Users/arizvi/.atom/packages/atom-speech-to-code/lib/speech-to-code.coffee:8:32)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:240:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:221:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:213:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:452:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:436:29
    at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:311:16)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:431:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:412:46)
    at Atom.module.exports.Atom.startEditorWindow (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:641:21)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:12:8)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:23:4)
    at Module._compile (module.js:452:26)
    at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
    at Module.load (module.js:347:32)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "vim-mode",
      "autocomplete"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
Sublime-Style-Column-Selection, v1.3.0
atom-handlebars, v0.5.0
atom-jshint, v2.0.0
atom-speech-to-code, v0.3.0
autocomplete-paths, v1.0.2
color-picker, v2.0.5
editorconfig, v1.0.0
emmet, v2.3.10
file-icons, v1.5.7
javascript-snippets, v1.0.0
jshint, v1.3.7
language-javascript-better, v1.5.0
linter, v0.12.7
monokai, v0.14.0
react, v0.12.0
white-cursor, v0.6.0

# Dev
No dev packages
```

**Comment by ikhmilevskiy** [comment]
having the same issue

**Comment by jedahan** [comment]
OK, let's see what I can do today to get it working...

**Comment by jedahan** [comment]
Should be fixed in b4bcaa67263a77b2ddac962739231f4144e435f2 / v0.4.0
Let me know if it is not fixed
